src/Sylius/Component/Shipping/Model/Shipment.php

```diff
@@ -22,6 +23,8 @@
  */
 class Shipment implements ShipmentInterface, TimestampableInterface
```

Comment:
I don't think it will be a major BC break because `Shipment` would still implement `TimestampableInterface`, but it'll be more consistent when typehinting objects as `ShipmentInterface` as it would include the created at and updated at dates. Now you can't access those dates on `ShipmentInterface`.

One and only BC would be if someone defined these methods on interface extending `ShipmentInterface`, but that's the time to break the compatability.